modules/nf-core/checkqc/main.nf

```diff
@@ -0,0 +1,48 @@
+process CHECKQC {
+    label 'process_low'
+    errorStrategy 'ignore'
```

Comment:
Is there a reason the default errorStrategy is to ignore errors?

Reply:
Yes, CheckQC returns a non-zero exit code when QC criteria aren't fulfilled. I was thinking that people using this module don't want the whole workflow to fail due to that. But it is probably better to let the user configure that behavior themselves. 

Another issue related to this is that the files from the process won't be published when CheckQC terminates with a non-zero exit code (even with `errorStrategy 'ignore'`). I was hoping it would, since the CheckQC report specifies QC criteria that weren't fulfilled. Do you have a suggestion on how to achieve this?

Reply:
You can do something like
```
checkQC ... || test -f "output_file"
```
Use the bash `||` so if checkqc fails, it will then run the `test` which can then test for something like a file that should be there even on failure ( otherwise use the sh `true` if a `test` isn't suitable ).

Reply:
Oh, thank you very much! I implemented that and added a test for a Failed QC result.